ManyWhoConstants.cs

```diff
@@ -373,6 +373,9 @@ public class ManyWhoConstants
         public static readonly Guid MANYWHO_FALSE_VALUE_ID = Guid.Parse("496FD041-D91F-48FB-AA4F-91C6C9A11CA1");
         public const string MANYWHO_FALSE_DEVELOPER_NAME = "$False";
 
+        public static readonly Guid MANYWHO_TENANT_ID_VALUE_ID = Guid.Parse("247AEE82-B65F-43AF-B28D-29E2458EF1E4");
+        public const string MANYWHO_TENANT_ID_DEVELOPER_NAME = "$TenantId";
```

Comment:
Once this is placed under the `$State` object, it should be named `Tenant ID` as there's no reason we need to mangle or change the capitalisation of names here.

Reply:
Sure, once it's under $State we won't need this any more either, so I'll cancel this PR

Reply:
Moved to `$State -> Tenant ID`